i18n/en_US.csv

```diff
@@ -111,3 +111,5 @@ Region,Region
 "API Token","API Token"
 "TaxJar Category Code","TaxJar Category Code"
 "Please redeploy production mode in the Magento CLI ($ bin/magento deploy:mode:set production) to ensure address validation works correctly.","Please redeploy production mode in the Magento CLI ($ bin/magento deploy:mode:set production) to ensure address validation works correctly."
+"Sandbox mode is enabled for sales tax calculations. We recommend only using this mode in a staging environment. Some features may not work as expected.","Sandbox mode is enabled for sales tax calculations. We recommend only using this mode in a staging environment. Some features may not work as expected."
+"Learn more","Learn more"
```

Comment:
@ryanreeves-taxjar Should we update `en_US.csv` to cover config descriptions / comments as well?